### PR TITLE
CI: Missing dependencies

### PR DESCRIPTION
**Pull request by me00001**
https://github.com/xlmnxp/blue-recorder/pull/67#issuecomment-2146908692

### CODE REVIEW

**Comment by xlmnxp** [comment]
thank you @me00001 @hamirmahal 🙏

**Comment by hamirmahal** [comment]
You're welcome @xlmnxp.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -11,5 +11,13 @@ jobs:
...
-            libunwind-dev
+            libunwind-dev \
+            librust-gstreamer-sys-dev \
+            librust-gdk-sys-dev \
+            librust-atk-dev \
+            librust-gtk4-sys-dev \
+            libgdk-pixbuf-2.0-dev \
+            librust-pango-dev \
+            librust-gobject-sys-dev \
+            ffmpeg
```
Comment (hamirmahal) [inline .github/workflows/test.yml:22]: This seems to work with just
```yml
          sudo apt-get install \
            librust-gdk-sys-dev \
            librust-gtk4-sys-dev \
            librust-gstreamer-sys-dev \
            libunwind-dev
```

**Comment by hamirmahal** [comment]
https://github.com/hamirmahal/blue-recorder/commit/b798069dccce5f10ea85749ef17f2c9ab1abcd84 is the commit that works with just

```yml
          sudo apt-get install \
            librust-gdk-sys-dev \
            librust-gtk4-sys-dev \
            librust-gstreamer-sys-dev \
            libunwind-dev
```

I will probably delete that branch soon. We may want to simplify the dependencies to just these four, to speed up CI workflow times and reduce code complexity.

**Pull request merged**